### DISCUSSION TITLE
Preview should be live and updated on filechange

### DISCUSSION THREAD

**Discussion by Jan-Eckerlein** [uxbox/uxbox#2461] [Ideas]
Hey, the only missing feature for me to ditch Figma is the live preview.
Currently, any changes in the editor are not reflected in the peview, unlike Figma.

This is important to me, because I want to see my changes' life on an actual Fullscreen

**Comment by niwinz** [comment]
Feel free to open an issue with this. GH discussions can be considered deprecated and don't expect we going to read them with the same priority as GH issues or Community post
